lib/bowling.rb

```diff
@@ -0,0 +1,37 @@
+class Bowling
+  attr_accessor :frames
+
```

Comment:
I think a more developed version of this might allow the user to simply enter their score after each roll, and then the program organises this into the array of arrays